### PR TITLE
[BXMSPROD-1980] drools-ansible integration github flow

### PR DESCRIPTION
**Pull request by Ginxo**
**JIRA**: https://issues.redhat.com/browse/BXMSPROD-1980

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* https://issues.redhat.com/browse/BXMSPROD-1980

### CODE REVIEW

**Comment by Ginxo** [comment]
I will revert @Ginxo
[temporary definition-file](https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/14/commits/7ce9466cf947abad9e93c791783d2fabc1d776ec) before merging

File: .ci/project-dependencies.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: "2.1"
+dependencies:
+  - project: kiegroup/drools
+    mapping:
+      dependencies:
+        default:
+          - source: (\d*)\.(.*)\.(.*)
+            targetExpression: "process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1-7}.${n2}.${n3}`)"
```
Comment (lampajr) [inline .ci/project-dependencies.yaml:8]: Is the mapping correct?

Comment (lampajr) [reply .ci/project-dependencies.yaml:8]: I was thinking something for drools like:

```
dependencies:
     default:
          - source: main
            target: main
```

and something for ansible-integration like:
```
dependencies:
     default:
          - source: main
            target: 8.35.x
```

Comment (Ginxo) [reply .ci/project-dependencies.yaml:8]: thanks @lampajr , let's wait for clarification from drools/ansible people @mariofusco 
ok?

**Review by bacciotti — APPROVED** [review]

File: .ci/project-dependencies.yaml (hunk 1 above)
Comment (Ginxo) [reply .ci/project-dependencies.yaml]: we agreed on having until versions and branches are clearer 
```
version: "2.1"
dependencies:
  - project: kiegroup/drools
    mapping:
      dependant:
        default:
          - source: 8.35.x
            target: 1.0.x

  - project: kiegroup/drools-ansible-rulebook-integration
    dependencies:
      - project: kiegroup/drools
    mapping:
      dependencies:
        default:
          - source: 1.0.x
            target: 8.35.x
```

**Review by lampajr — APPROVED** [review]

File: .ci/pull-request-config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+
+dependencies: ./project-dependencies.yaml
+
+build:
+  - project: kiegroup/drools
+    build-command:
+      current: mvn --batch-mode --update-snapshots install -Dquickly
```
Comment (mareknovotny) [inline .ci/pull-request-config.yaml:8]: @mariofusco can you check the maven command?

File: .ci/pull-request-config.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  - project: kiegroup/drools-ansible-rulebook-integration
+    build-command:
+      current: mvn --batch-mode --update-snapshots verify
```
Comment (mareknovotny) [inline .ci/pull-request-config.yaml:12]: @mariofusco again this is for review of maven command

**Review by mariofusco — APPROVED** [review]

**Review by mareknovotny — APPROVED** [review]

**Review by rgdoliveira — APPROVED** [review]

**Pull request merged**